pkg/signup/service/signup_service.go

```diff
@@ -279,7 +280,7 @@ func (s *ServiceImpl) GetSignup(userID string) (*signup.Signup, error) {
 		Ready:                ready,
 		Reason:               murCondition.Reason,
 		Message:              murCondition.Message,
-		VerificationRequired: userSignup.Spec.VerificationRequired,
+		VerificationRequired: states.VerificationRequired(userSignup),
```

Comment:
Should be 
```suggestion
		VerificationRequired: states.VerificationRequired(userSignup) || userSignup.Spec.VerificationRequired,
```
right?